src/ui/common/src/components/workflows/nodes/MetricOperatorNode.tsx

```diff
@@ -69,17 +75,49 @@ const MetricOperatorNode: React.FC<Props> = ({ data, isConnectable }) => {
       ? theme.palette.DarkWarningMain50
       : theme.palette.DarkWarningMain;
     hoverColor = theme.palette.DarkWarningMain75;
+
+    displayValue = (
```

Comment:
Could you make this and the other icon into their own components? I see that the same code is being used several times in this file. You can just set displayValue to those components when doene.